patches/minecraft/net/minecraft/entity/passive/HorseArmorType.java.patch

```diff
@@ -0,0 +1,62 @@
+--- ../src-base/minecraft/net/minecraft/entity/passive/HorseArmorType.java
++++ ../src-work/minecraft/net/minecraft/entity/passive/HorseArmorType.java
+@@ -25,6 +25,14 @@
+         this.field_188587_f = "";
+     }
+ 
++    //Allows for textures located outside the vanilla horse armor folder
++    private  HorseArmorType(String textureLocation, String hash, int armorStrengthIn) 
```

Comment:
Ok, but what's wrong with private? No one has any business accessing that ctor unless it's through the EnumHelper method. Is there a Java convention or Forge convention to use the highest access level you can use without unnecessarily exposing classes/fields/methods?